usaspending_api/search/v2/views/spending_by_subaward_grouped.py

```diff
@@ -0,0 +1,151 @@
+import copy
+import logging
+from sys import maxsize
+from typing import Any
+from dataclasses import dataclass
+from decimal import Decimal
+from django.conf import settings
+from elasticsearch_dsl import A
```

Comment:
Won't block over this, but would be good to combine the two `elasticsearch_dsl` imports into one line. Realizing now how many places across the codebase we split this up for some reason.